unified-scheduler-pool/src/lib.rs

```diff
@@ -1325,13 +1311,13 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
 
     fn start_session(
```

Comment:
now `start_session()` and `spawn()`'s arg types are aligned as much as possible.